src/QuantumSdk/DefaultItems/DefaultItems.targets

```diff
@@ -107,17 +115,24 @@
       <_ResolvedTargetPackageReference Include="$(%(_TargetPackageReferencePathProperty.Identity))" />
       <ResolvedTargetSpecificDecompositions Include="@(_ResolvedTargetPackageReference->Split(`;`))" />
     </ItemGroup>
-    <Error Condition="@(ResolvedTargetSpecificDecompositions->Count()) &gt; 1" Text="More than one target package was defined. The defined target package needs to be unique." />
```

Comment:
I randomly came across that the Sdk prevents that more than more than one dll can contain target specific decompositions. The compiler supports multiple dlls - I believe I merely forgot to remove this here.

Reply:
The compiler should actually generate an error in this case; i.e. the ordering is not hierarchical and the expectation is that there are declarations within all target package dlls are unique.